collector/src/compile/execute/profiler.rs

```diff
@@ -51,6 +52,68 @@ impl Profiler {
                 | Profiler::DepGraph
         )
     }
+
+    pub fn diff_prereqs(&self) -> anyhow::Result<()> {
+        use Profiler::*;
+        match self {
+            Cachegrind => {
+                check_installed("valgrind")?;
+                check_installed("cg_annotate")?;
+            }
+
+            SelfProfile | PerfRecord | Oprofile | Samply | Callgrind | Dhat | DhatCopy | Massif
+            | Bytehound | Eprintln | LlvmLines | MonoItems | DepGraph | LlvmIr => {}
+        }
+        Ok(())
+    }
+
+    /// A file prefix added to all files of this profiler.
+    pub fn prefix(&self) -> &'static str {
+        use Profiler::*;
+        match self {
+            Cachegrind => "cgout",
+            DepGraph => "dep-graph",
+
+            SelfProfile | PerfRecord | Oprofile | Samply | Callgrind | Dhat | DhatCopy | Massif
+            | Bytehound | Eprintln | LlvmLines | MonoItems | LlvmIr => "",
+        }
+    }
+
+    /// A postfix added to the file that gets diffed.
+    pub fn postfix(&self) -> &'static str {
```

Comment:
Is the `.txt` postfix actually important for anything? If not, we could just get rid of this method.

Reply:
Yea, differentiating it from the `.dot` file. Other profilers have multiple outputs, too. I wanna make this return a `Vec` in the future to diff all the individual pieces. Graphviz can produce diff graphs iirc

Reply:
Ok :)